content/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -409,3 +409,11 @@ curl \
   --request DELETE \
   https://app.terraform.io/api/v2/oauth-clients/oc-XKFwG6ggfA9n7t1K
 ```
+
+### Available Related Resources
+
+The GET endpoints above can optionally return related resources, if requested with [the `include` query parameter](/cloud-docs/api-docs/#inclusion-of-related-resources). The following resource types are available:
+
+| Resource Name  | Description                              |
+| -------------- | ---------------------------------------- |
+| `oauth-tokens` | The OAuth tokens managed by this client  |
```

Comment:
I think the field is actually "oauth_tokens" with an underscore. I don't think we do any kind of dasherizing on these values

Reply:
Yeah, Brandon is (unfortunately!) correct! Although we use `dasherized-case` (kebab case? I can never remember) within the API path names, we use `snake_case` in the `include` parameter. :coolcrying: